iDiff/differs/history.go

```diff
@@ -0,0 +1,43 @@
+package differs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docker/docker/client"
+	"github.com/pmezard/go-difflib/difflib"
```

Comment:
You should setup golang folder vendoring ('/vendor') and potentially use a package manager like godeps (although that might be overkill) if you plan on using third party libraries.